Update user-manual.md - Settings Screen (#178)
sspanak/tt9: docs/user-manual.md

## Settings Screen
On the Settings screen, you can choose languages for typing, configure the keypad hotkeys or change the application appearance.

### How to access the Settings?

#### Method 1
- Start typing in a text field to wake up TT9.

> - Use the on-screen gear button or press the Settings Key.

Reviewer: ```suggestion
- Use the on-screen gear button.
```

On the Settings screen, you can choose languages for typing, configure the keypad hotkeys or change the application appearance.

> To access it:

Reviewer: ```suggestion
### How to access the Settings?
```

#### Method 1
- Start typing in a text field to wake up TT9.
- Use the on-screen gear button or press the Settings Key.

#### Method 2

> - Press the Assigned Hotkey [Default: Hold *]

Reviewer: ```suggestion
- Start typing in a text field to wake up TT9.
- Use the assigned hotkey [Default: Hold ✱].
```

#### Method 3

> - Thru Android Settings: System -> Languages -> (On-screen) Keyboard(s) -> Traditional T9.

Reviewer: Let's extend this a bit:

```suggestion
- Go to Android Settings → System → Languages → Keyboards (or On-Screen Keyboards). This is where all installed keyboards are configured.
- Select "Traditional T9".

_The actual menu names may vary depending on your phone, Android version and language._
```
